src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/" class="logo"><img src="/moyo_1.svg" alt="logo"></a>
```

Comment:
[Checklist](https://github.com/Rusik-best86/layout_moyo-header/blob/develop/checklist.md):
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/137c6759-8320-4a63-a31f-0e4527872ee9)
- And fix formatting, don't put nested elements in the parent line